pkg_security.Rmd

```diff
@@ -1,19 +1,73 @@
 # Package Development Security Best Practices
 
 ```{block, type='summaryblock'}
-This chapter isn't written yet. It should mention the elements outlined in [this bookdown](https://ropenscilabs.github.io/r-security-practices/) ([repo](https://github.com/ropenscilabs/r-security-practices)) started at the unconf 2017, or have a link to this book/a similar book when written.
+This chapter isn't complete yet. It includes [guidance about managing secrets in packages](#pkgsecrets) and [links for further reading](#furthersecreading).
 ```
 
-In the meantime, here are useful security resources:
+## Misc
 
-* [rOpenSci community call "Security for R"](https://ropensci.org/commcalls/2019-05-07/) (recording, slides, etc. see in particular [the list of resources](https://ropensci.org/blog/2019/04/09/commcall-may2019/#resources));
+We recommend you [secure your GitHub account with 2FA](https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/).
 
-* [the security-related projects of unconf18](https://ropensci.org/blog/2018/06/06/unconf18_recap_2/);
+If the web service your package wraps has either https or http, opt for https.
 
-* [the `notary` package](https://ropensci.org/blog/2017/07/25/notary/).
+## Secrets in packages {#pkgsecrets}
 
-## Misc
+Guidance for when you develop a package interacting with a web resources asking for credentials (API keys, tokens, etc.). Also refer to [the `httr` vignette about sharing secrets](https://httr.r-lib.org/articles/secrets.html).
 
-We recommend you [secure your GitHub account with 2FA](https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/).
+### Secrets in packages and user protection
+
+Say your package needs an API key for making requests on behalf of your user.
+
+* In your package documentation, try to make sure the API key won't end up in the .Rhistory/script of your user.
```

Comment:
"guide the user, so the API key" or similar instead of "try to make sure the API key"